README.md

```diff
@@ -241,48 +135,32 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
+## ⭐️ Show your support <a name="support"></a>
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+If you like this project, kindly contact me on my github: Authors](#authors)
```

Comment:
- [ ] In this section should convey a request to the visitors about staring your project in case they love it, so Kindly fix what you are trying to ask from the visitors 👍🏼